static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -0,0 +1,464 @@
+          or dragging the bar above. The panel can be hidden or shown by clicking the
+          handle or using the
+          {' '}
+          <kbd>]</kbd>
+          {' '}
+          key.
+        </p>
+        <div className="tour-info-box">
+          <p>
+            All keyboard shortcuts can be displayed by pressing the
+            {' '}
+            <kbd>?</kbd>
+            {' '}
+            key at any time.
+          </p>
+        </div>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'right',
+    styles: commonStyles
+  },
+  {
+    target: '.right-overlay',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          Pan the map by clicking and dragging, and zoom by using the scroll wheel or
+          map tools.
+        </p>
+        <p className="tour-content">
+          When a collection is selected, the granules will be displayed on the map,
+          along with any available GIBS imagery. When a granule is focused on the
+          map, any additional thumbnails will be displayed.
+        </p>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'left',
+    styles: commonStyles
+  },
+  {
+    target: '.leaflet-bottom.leaflet-right',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          Use the map tools to switch map projections, draw, edit, or remove spatial
+          bounds, zoom the map, or select the base map.
+        </p>
+        <TourButtons stepIndex={stepIndex} setStepIndex={setStepIndex} />
+      </div>
+    ),
+    placement: 'left',
+    styles: commonStyles
+  },
+  {
+    target: '.secondary-toolbar__start-tour-button',
+    content: (
+      <div>
+        <StepCounter currentStep={stepIndex} />
+        <p className="tour-content">
+          You can replay this tour anytime by clicking
+          {' '}
+          <strong>Show Tour</strong>
+          .
+        </p>
+        <div className="tour-buttons">
+          <Button
+            type="button"
+            bootstrapVariant="secondary"
+            bootstrapSize="sm"
+            onClick={
+              () => {
+                setStepIndex(stepIndex - 1)
+              }
+            }
+          >
+            Previous
+          </Button>
+          <Button
+            className="button-tour-finish"
+            type="button"
+            bootstrapVariant="primary"
+            bootstrapSize="sm"
+            onClick={
+              () => {
+                setStepIndex(stepIndex + 1)
+              }
+            }
+          >
+            Finish Tour
+          </Button>
+        </div>
+      </div>
+    ),
+    placement: 'right',
+    styles: commonStyles
+  },
+  {
+    target: '.search',
+    content: (
+      <div>
+        <h2 className="tour-heading">
```

Comment:
This step doesnt quite match the figma document. 

Here are the things that look off:
- The heading font is incorrect.
- The spacing below the heading is too small
- The watch the webinar link looks incorrect
- There is extra space between the "Find more information here:" links
- There is extra space below the "Find more information here:" section